server/initialize.go

```diff
@@ -237,6 +239,14 @@ func ensureMySQLUser(ctx context.Context, db *sqlx.DB, user UserSetting, pwd str
 		return fmt.Errorf("failed to grant to %s: %w", user.name, err)
 	}
 
+	if user.proxyAdmin {
+		queryStr = `GRANT PROXY ON ''@'' TO ?@'%' WITH GRANT OPTION`
+		_, err = db.ExecContext(ctx, queryStr, user.name)
+		if err != nil {
```

Comment:
Just as a suggestion.  We often write this as:
```suggestion
		if _, err := db.ExecContext(ctx, queryStr, user.name); err != nil {
```